README.md

```diff
@@ -58,27 +58,40 @@ You can do that in your `config/database.php` file, where you define your databa
 
 ```php
 'mysql' => [
-            'driver'            => 'mysql',
-            'host'              => env('DB_HOST', 'localhost'),
-            'database'          => env('DB_DATABASE', 'forge'),
-            'username'          => env('DB_USERNAME', 'forge'),
-            'password'          => env('DB_PASSWORD', ''),
-            'charset'           => 'utf8',
-            'collation'         => 'utf8_unicode_ci',
-            'prefix'            => '',
-            'strict'            => false,
-            'engine'            => null,
-            'dump' => [
- 
-               'dump_binary_path' => '/path/to/directory/', // only the path, without `mysqldump` or `pg_dump`
-               // 'dump_binary_path' => '/Applications/MAMP/Library/bin/', // works for MAMP on Mac OS
-               // 'dump_binary_path' => '/opt/homebrew/bin/', // works for Laravel Valet on Mac OS
-               'use_single_transaction',
-               'timeout' => 60 * 5, // 5 minute timeout
-               // 'exclude_tables' => ['table1', 'table2'],
-               // 'add_extra_option' => '--optionname=optionvalue',
-            ]
-        ],
+    'driver'    => 'mysql',
+    'host'      => env('DB_HOST', 'localhost'),
+    'database'  => env('DB_DATABASE', 'forge'),
+    'username'  => env('DB_USERNAME', 'forge'),
+    'password'  => env('DB_PASSWORD', ''),
+    'charset'   => 'utf8',
+    'collation' => 'utf8_unicode_ci',
+    'prefix'    => '',
+    'strict'    => false,
+    'engine'    => null,
+    'dump' => [
+        'dump_binary_path' => '/path/to/directory/', // only the path, without `mysqldump` or `pg_dump`
+        // 'dump_binary_path' => '/Applications/MAMP/Library/bin/', // works for MAMP on Mac OS
+        // 'dump_binary_path' => '/opt/homebrew/bin/', // works for Laravel Valet on Mac OS
+        'use_single_transaction',
+        'timeout' => 60 * 5, // 5 minute timeout
+        // 'exclude_tables' => ['table1', 'table2'],
+        // 'add_extra_option' => '--optionname=optionvalue',
+    ]
+],
+```
+
+BackupManager uses Livewire, if you use it too in any costumization to your admin panel you may face some Javascript issues, please use the below code snipped to avoid importing livewire multiple times:
```

Comment:
```suggestion
BackupManager uses [Livewire](https://laravel-livewire.com/). If you use it in your admin panel too, and its Javascript & CSS is already onpage, you may face some Javascript issues. If you're using a custom layouts that already loads Livewire, please use the code below. This will avoid loading Livewire assets multiple times in one page:
```